### DISCUSSION TITLE
installing docker image on fly io server

### DISCUSSION THREAD

**Discussion by DavidMitchell89** [TriliumNext/Trilium#4565] [Q&A]
I spent some time playing with a local install of the app and I really like it, so now I'm attempting to install it on a fly server so I can access it when I'm out of the house and struggling to get it to stay persistent.

I pulled the docker image and then ran the command
`fly launch --no-deploy -i zadam/trilium

I then edited the fly toml to give the app a persistent volume, if I navigate to the apps url it launches fine and I can setup the app but if i leave and come back the app is reset. 

My google skills have failed me and I'm not sure what command I'm missing to make the app persistent. is anyone able to point me in the right direction?

**Comment by sigaloid** [comment]
Fly.io will automatically spin down the server and scale down to zero to save on costs. Look at your fly.toml and set the minimum machine count to 1

**Reply by DavidMitchell89** [reply]
yep, that fixed it. thank you!
